apis/tempo/v1alpha1/tempostack_webhook.go

```diff
@@ -333,6 +333,13 @@ func (v *validator) validateObservability(tempo TempoStack) field.ErrorList {
 		}
 	}
 
+	if tempo.Spec.Observability.Grafana.CreateDatasource && !v.ctrlConfig.Gates.GrafanaOperator {
+		return field.ErrorList{
+			field.Invalid(metricsBase.Child("createDatasource"), tempo.Spec.Observability.Grafana.CreateDatasource,
```

Comment:
```suggestion
			field.Invalid(grafanaBase.Child("createDatasource"), tempo.Spec.Observability.Grafana.CreateDatasource,
```

Before the condition, declare
```
grafanaBase := observabilityBase.Child("grafana")
```

Reply:
Updated, I missed that 😶‍🌫️